shell/app/atom_content_client.cc

```diff
@@ -155,7 +156,7 @@ void ComputeBuiltInPlugins(std::vector<content::PepperPluginInfo>* plugins) {
   pdf_info.is_out_of_process = true;
   pdf_info.name = "Chromium PDF Viewer";
   pdf_info.description = "Portable Document Format";
-  pdf_info.path = base::FilePath(FILE_PATH_LITERAL("internal-pdf-viewer"));
+  pdf_info.path = base::FilePath(FILE_PATH_LITERAL("internal-pdf-viewer2"));
```

Comment:
This definitely needs to be cleaned up, I'll follow Chrome a little more closely when I come back to refactor this.